setup.py

```diff
@@ -1,5 +1,5 @@
 # This file is part of Indico.
-# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
```

Comment:
Let's remove that whole header. It's the old GPL one from a LONG time ago and completely useless in this particular repo... there's literally no actual code besides this setup.py (and I'd move all this into a setup.cfg tbh like we do in the other packages)

Reply:
i'd also bump the version to 1.2